spiffworkflow-backend/src/spiffworkflow_backend/__init__.py

```diff
@@ -184,6 +184,10 @@ def create_app() -> flask.app.Flask:
     app.before_request(AuthorizationService.check_for_permission)
     app.after_request(set_new_access_token_in_cookie)
 
+    # The default is true, but we want to preserve the order of keys in the json
+    # This is particularly helpful for forms that are generated from json schemas.
+    app.json.sort_keys = False
```

Comment:
i created a process model and the keys are still sorted, so that's good. i think we started sorting those keys so they would stop being "randomized" when different people clicked save on process groups and models. i imagine we have specific code for these cases and it is therefore unaffected by this change.

Reply:
Shot in the dark but is there any chance this fixes https://github.com/sartography/spiff-arena/issues/476

Reply:
It helps, but the python introspection on parameters it's offering any promises on order - so it won't alphabetize them anymore, but they will still be random.